src/Umbraco.Core/StringExtensions.cs

```diff
@@ -1,1602 +1,1602 @@
-        /// </summary>
-        /// <param name="haystack">The string to check</param>
-        /// <param name="needles">The collection of strings to check are contained within the first string</param>
-        /// <param name="comparison">The type of comparision to perform - defaults to <see cref="StringComparison.CurrentCulture"/></param>
-        /// <returns>True if any of the needles are contained with haystack; otherwise returns false</returns>
-        /// Added fix to ensure the comparison is used - see http://issues.umbraco.org/issue/U4-11313
-        public static bool ContainsAny(this string haystack, IEnumerable<string> needles, StringComparison comparison = StringComparison.CurrentCulture)
-        {
-            if (haystack == null)
-                throw new ArgumentNullException("haystack");
-
-            if (string.IsNullOrEmpty(haystack) || needles == null || !needles.Any())
-            {
-                return false;
-            }
-
-            return needles.Any(value => haystack.IndexOf(value, comparison) >= 0);
-        }
-
-        public static bool CsvContains(this string csv, string value)
-        {
-            if (string.IsNullOrEmpty(csv))
-            {
-                return false;
-            }
-            var idCheckList = csv.Split(new[] { "," }, StringSplitOptions.RemoveEmptyEntries);
-            return idCheckList.Contains(value);
-        }
-
-        // From: http://stackoverflow.com/a/961504/5018
-        // filters control characters but allows only properly-formed surrogate sequences
-        private static readonly Regex InvalidXmlChars =
-            new Regex(
-                @"(?<![\uD800-\uDBFF])[\uDC00-\uDFFF]|[\uD800-\uDBFF](?![\uDC00-\uDFFF])|[\x00-\x08\x0B\x0C\x0E-\x1F\x7F-\x9F\uFEFF\uFFFE\uFFFF]",
-                RegexOptions.Compiled);
-
-
-        /// <summary>
-        /// An extension method that returns a new string in which all occurrences of an 
-        /// unicode characters that are invalid in XML files are replaced with an empty string. 
-        /// </summary>
-        /// <param name="text">Current instance of the string</param>
-        /// <returns>Updated string</returns>
-        /// 
-        /// <summary>
-        /// removes any unusual unicode characters that can't be encoded into XML
-        /// </summary>
-        internal static string ToValidXmlString(this string text)
-        {
-            return string.IsNullOrEmpty(text) ? text : InvalidXmlChars.Replace(text, "");
-        }
-
-        /// <summary>
-        /// Converts a string to a Guid - WARNING, depending on the string, this may not be unique
-        /// </summary>
-        /// <param name="text"></param>
-        /// <returns></returns>
-        internal static Guid ToGuid(this string text)
-        {
-            return CreateGuidFromHash(UrlNamespace,
-                                        text,
-                                        Cryp
```

Comment:
The change happens here.

Reply:
Hi @elit0451 

I think this will break the strip method... See my comment in PR #3818